### PR TITLE
Ws 130 nft create nft and change nft documentation

### PR DESCRIPTION
**Pull request by fedeaviles**
**Type of MR:**

- [x] Feature
- [ ] Fix
- [ ] HotFix

**Small description:**
Add docs for:
- NFTImageFactory 
- NFTJsonFactory
- NFTJsonImageFactory
- NFTHandler

# Checklist:

- [x] My code follows the styleguides of the project (successfully pass linters)
- [x] I tested my code.
- [ ] I commented/docummented my code in a way that complex parts are explained.
- [x] If required, i updated project's README file.
- [ ] I added/updated tests that tests my changes in code.
- [x] All previous and new tests pass.
- [ ] I commit any new dependency.
- [ ] I updated UML diagram in confluence if any model was modified in this MR.

### CODE REVIEW

**Review by deivrios — APPROVED** [review]

File: docs/sdk-nft/README.md [hunk 1]
```diff
@@ -62,33 +62,151 @@ nft.jsonData;
...
+
+Static method that, given a Resource (which returns an image) and a tokenId, it returns an NFT instance.
+
+```javascript
+// commonJS
+const { NFTImageFactory } = require('@rgc/sdk');
+// TS
+import { NFTImageFactory } from '@rgc/sdk';
+
+NFTImageFactory.createNFT(RESOURCE, "aTokenId");
```
Comment (AgusVelez5) [inline docs/sdk-nft/README.md:77]: Should we add the rest of the code for the user to copy, paste and run? (function with async and await but obviously without the nft data)

Just an example:
```
const m = async () => {
    const nft = await NFTImageFactory.createNFT(RESOURCE, "aTokenId");
}
m()
```

And the same with the following examples

Comment (fedeaviles) [reply docs/sdk-nft/README.md:77]: You are right. I'm just going to add de `await`, like the rest of the docs

**Review by AgusVelez5 — APPROVED** [review]

File: docs/sdk-nft/README.md [hunk 2]
```diff
@@ -62,33 +62,151 @@ nft.jsonData;
...
+- get:
 
+Get an NFT for a token id.
+```javascript
+await NFTHandler.get(NFT_MANAGER, STORAGE, "aTokenId", FORMAT);
+NFT {...} // NFT instance
+```
 Where:
 
 - NFT_MANAGER is a NFT721Manager instance activated
```
Comment (mauricio-b-r) [inline docs/sdk-nft/README.md:142]: NFT_MANAGER is an activated NFT721Manager instance

File: docs/sdk-nft/README.md [hunk 3]
```diff
@@ -62,33 +62,151 @@ nft.jsonData;
...
+    TOKENID,
+    NFTMETADATA,
+    IMAGE,
+    VALUE
+};
+await NFTHandler.change(params); // returns a ExecuteResponse
 ```
+Where:
+
+- NFT_MANAGER is a NFT721Manager instance activated
```
Comment (mauricio-b-r) [inline docs/sdk-nft/README.md:203]: NFT_MANAGER is an activated NFT721Manager instance

**Pull request merged**